internal/error_test.go

```diff
@@ -1245,3 +1245,44 @@ func Test_convertFailureToError_SaveFailure(t *testing.T) {
 	require.Equal("SomeJavaException", f2.GetCause().GetApplicationFailureInfo().GetType())
 	require.Equal(true, f2.GetCause().GetApplicationFailureInfo().GetNonRetryable())
 }
+
+func Test_verbose_error_formatting(t *testing.T) {
+	tests := []struct {
+		name     string
+		err      error
+		expected string
+	}{
+		{
+			name:     "WorkflowExecutionError",
+			err:      NewWorkflowExecutionError("wid", "rid", "workflowType", newWorkflowPanicError("test message", "stack trace")),
+			expected: "stack trace\ntest message\nworkflow execution error (type: workflowType, workflowID: wid, runID: rid): test message",
```

Comment:
It seems odd to show inner error information before outer error information

Reply:
I basically copied the behavior from github.com/pkg/errors - I did also think the same but for consistency it seemed like the best approach